source/PyMaterialX/CMakeLists.txt

```diff
@@ -38,6 +38,14 @@ if(CMAKE_CXX_COMPILER_ID MATCHES Clang)
         add_compile_options(-Wno-undefined-var-template)
     endif()
 endif()
+
+# Postfix rules for Python modules are strict and depend on the presence of a
+# debug interpreter:
+if(PYTHON_IS_DEBUG)
+    set(MATERIALX_PYTHON_DEBUG_POSTFIX "_d")
```

Comment:
Debug interpreter, using python27_d.lib

Reply:
Is this also the case for Python3 and on other platforms?

Reply:
Confirmed for Python 2.7 version up to 3.1, especially on Windows:
https://github.com/python/cpython/blob/c4cacc8c5eab50db8da3140353596f38a01115ca/PCbuild/readme.txt#L39

Has become a lot more complex following Python 3.2 (or actually a lot simpler by calling `sysconfig.get_config_var('EXT_SUFFIX')`):
Details: https://www.python.org/dev/peps/pep-3149/
Boost::python is aware: https://github.com/boostorg/build/issues/321
Does not appear in PyBind11's issue tracker yet.

And with 3.8 debug and release binaries can coexist peacefully: 
https://docs.python.org/3/whatsnew/3.8.html#debug-build-uses-the-same-abi-as-release-build

Reply:
Thanks @ashwinbhat for asking the right question. I ran tests and PyBind11 implements PEP 3149 for Python 3 perfectly on all test platforms. I made sure the suffix is only used for legacy Python 2.7 when a Debug interpreter is found, on the Windows platform.